scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/GetSpecificationRulesAction.java

```diff
@@ -0,0 +1,59 @@
+package gov.nasa.jpl.aerie.scheduler.server.remotes.postgres;
+
+import gov.nasa.jpl.aerie.scheduler.server.models.SpecificationId;
+import org.intellij.lang.annotations.Language;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+/*package-local*/ final class GetSpecificationRulesAction implements AutoCloseable {
```

Comment:
I'm still on the fence on the naming (and I will be for a long time, so no need to wait for me!)

However, for the sake of consistency with your other code - this should probably be renamed `GetSpecificationGoalsAction`.

Reply:
Absolutely, I just missed it while renaming things! Thank you for your careful examination!

Reply:
Resolving as this was a simple update and don't want to get held up on this not being marked resolved.